robot_cfg:
  name: xarm7
  dof_num: 7
  # NOTE: must match Isaacâ€™s articulation joint order
  joint_names: ["joint1","joint2","joint3","joint4","joint5","joint6","joint7"]

  urdf_path: "~/revo/egox/xarm7/xArm7_teleop/assets/xarm7_with_gripper.urdf"

  ee: "gripper"    # CHECK: confirm if this or link7

  # TODO: to be tuned. taken from init config of xarm7 in Isaac Sim
  arm_init: [0.435, -0.17, 2.085, 1.018, 0.0, 0.0, 0.801]

  # 7 joints
  arm_indices: [0,1,2,3,4,5,6]

  # TODO: to be tuned and tested
  wrist_low_pass_alpha: 0.15

  # IK controller parameters: used same as pinocchio test
  arm:
    out_lp_alpha: 0.5
    damp: 1e-12
    eps: 1e-4
    dt: 1e-1
